AK/StdLibExtras.h

```diff
@@ -68,6 +68,13 @@ constexpr SizeType array_size(T (&)[N])
     return N;
 }
 
+template<typename T>
+requires requires { Detail::Decay<T> {}.size(); }
```

Comment:
Why are there two `requires` here?

Reply:
`requires requires`
https://quuxplusone.github.io/blog/2019/01/15/requires-requires-is-like-noexcept-noexcept/

Reply:
Thanks, C++ is ridiculous sometimes but 🤷